examples/spec/cassandra.ini

```diff
@@ -0,0 +1,1978 @@
+[]
+mountpoint = cassandra.ini
+infos/plugins = dump
+
+
+[/cluster_name]
```

Comment:
beginning slash?

Reply:
Does is make a difference? actually it doesn't affect the code but I can remove it

Reply:
It is style, so it makes a pragmatic difference.

Unfortunately the style is not described in doc/CODING.md yet but you can see the other examples.